src/sandstorm/supervisor.capnp

```diff
@@ -154,7 +158,7 @@ struct SupervisorObjectId(AppObjectId) {
     appRef @0 :AppObjectId;
     # A reference restorable by the app.
 
-    wakeLockNotification @1 :UInt32;
+    wakeLockNotification @1 :Data;
```

Comment:
This was a UInt32 before because I figured you'd just use a counter. SupervisorObjectId is allowed to contain guessable content, since it will be suitably obscured behind an ApiToken.